src/coredata.cs

```diff
@@ -127,6 +137,9 @@ public interface NSEntityDescription : NSCoding, NSCopying {
 		[Export ("managedObjectClassName")]
 		string ManagedObjectClassName { get; set; }
 
+		[Export ("renamingIdentifier")]
+		string RenamingIdentifier { get; set; }
```

Comment:
Pasting the gutter conversation:

Chris Hamons @chamons 18:15
well so that works, but that isn't exactly what i mean
in some corners of Cocoa, you have to pass in a "known" identifier
and so you go myThing = SomeGlobalThing
and as opposed to actually doing a string compare, they just compare the pointer of that global to you identifier
so if you round trip NSString -> C# string -> NSString it doesn't work 
but identifier seems ok, i don't see any constants you need to shove in
(don't know this API)

Manuel de la Pena @mandel-macaque 18:17
@chamons the API does not work like that, you are passing the ewnaming of a property, so you have model X, witha property Y, then you update the object to rename Y to Z and that RenamingProperty allows to do the match

Chris Hamons @chamons 18:17
then you are safe and get my 

Manuel de la Pena @mandel-macaque 18:17
@chamons is like, hey this is a property that used to be called Manuel, but now is Pedro